packages/core/src/routes/well-known.ts

```diff
@@ -46,30 +65,12 @@ export default function wellKnownRoutes<T extends AnonymousRouter>(router: T, pr
         ];
       }, []);
 
-      // Hard code AdminConsole sign-in methods settings.
-      if (interaction?.params.client_id === adminConsoleApplicationId) {
-        ctx.body = {
-          ...signInExperience,
-          signInMethods: adminConsoleSignInMethods,
-          signInMode: (await hasActiveUsers()) ? SignInMode.SignIn : SignInMode.Register,
-          socialConnectors: [],
-        };
-
-        return next();
-      }
-
-      // Insert Notification Message to DemoApp
-      if (interaction?.params.client_id === demoAppApplicationId) {
-        ctx.body = {
-          ...signInExperience,
-          socialConnectors,
-          notification: i18next.t('demo_app.notification'),
-        };
-
-        return next();
-      }
+      const notification =
+        interaction?.params.client_id === demoAppApplicationId
```

Comment:
nit: maybe better extract `interaction?.params.client_id === demoAppApplicationId` to a local variable for reuse here and above.

```ts
      // Hard code AdminConsole sign-in methods settings.
      if (interaction?.params.client_id === adminConsoleApplicationId) {
```